app/views/admin/users/_member_index.html.erb

```diff
@@ -7,34 +7,64 @@
 
     <!-- Small screen data view start -->
     <div class="block m:hidden fs-s">
-      <h2 class="member-data-heading fs-s py-2 -mx-4 px-6 color-base-60">Members</h2>
-      <ul class="list-none mx-2">
-        <% @users.each do |user| %>
+      <h2 class="member-data-heading fs-s py-2 -mx-3 px-3 color-base-60">Members</h2>
+      <ul class="list-none">
+        <% @users.includes([:organization_memberships]).each do |user| %>
           <li class="py-4">
             <article>
               <header class="flex">
-                <img class="radius-full mr-2" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="" />
-                <div class="flex flex-col">
-                  <h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
-                  <span class="color-base-60">@<%= user.username %></span>
+                <a href="<%= admin_user_path(user.id) %>">
+                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                </a>
+                <div class="pl-3 flex-1 fs-s"><%= link_to user.name, admin_user_path(user) %>
```

Comment:
```suggestion
                <div class="pl-3 flex-1 fs-s"><h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
```
Ideally, it would great if the user name could still be a `h3` heading, as it marks out the whole section for screen reader users to be able to jump to (since we're not in a table in this view, it could be quite difficult to navigate the rows of content)

Reply:
Fair enough! I've reverted this change. 😊